content/en/user-guide/integrations/quarkus/index.md

```diff
@@ -0,0 +1,156 @@
+---
+title: "Quarkus"
+weight: 11
+description: >
+  Use the Quarkus framework with LocalStack
+---
+
+Quarkus is a Java framework optimized for cloud, serverless, and containerized environments. Quarkus leverages a Kubernetes Native Java stack tailored for GraalVM & OpenJDK HotSpot, which further builds on various Java libraries and standards.
+
+Localstack is supported by Quarkus as a Dev service for Amazon Services. Quarkus Amazon Services automatically starts a LocalStack container in development mode and when running tests, and the extension client is configured automatically.
+
+{{< alert title="Note" >}}
```

Comment:
nit: seems like quite a specific detail - this is the first mention of `pom.xml` in the text and at this point the user doesn't even know about the Maven setup yet. Perhaps it could be moved a bit further down the page?

Reply:
Sure, will do 👍